DocumentedExamples/regional-mom6-forced-by-access-om2.ipynb

```diff
@@ -0,0 +1,1238 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    buffer = 0.2 # degrees
</pre><p>This should be at the top, and the 0.2 degrees in the top 'paragraph' also replaced with buffer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
nice catch!